### DISCUSSION TITLE
Database tables setup instruction

### DISCUSSION THREAD

**Discussion by CrazyMoka** [quartznet/quartznet#1631] [Q&A] [answered]
Forgive me if its in the documentation but, I can't seem to find how to setup the tables for quartz.
We are using mssql. I can see code snippits to add it to the config, but I do not see anywhere about where the tables are how to add them and how to test that it worked.

Please advise

Thank you

**Comment by ghost** [comment, marked as answer]
When you download and unzip the source code, they are in the quartznet-main\quartznet-main\database\tables folder. You will want the tables_sqlServer.sql file for MS SQL Server.

**Reply by ghost** [reply]
Or on github, here:  https://github.com/quartznet/quartznet/tree/main/database/tables

**Reply by lahma** [reply]
Exactly as @BrentHet described. Should definitely make this more prominent on web site.

**Reply by CrazyMoka** [reply]
We are using dotnet code Entity Framework. Using code first for models.
Will this impact anything other then if I want to access things I'll have to do raw queries?

**Reply by lahma** [reply]
@mhebel3 you should *NOT* access Quartz tables directly, you can cause locking issues and all sorts of unexpected behavior. Quartz data should be accessed via `IScheduler` interface, you can configure a scheduler with `ZeroSizeThreadPool` and not starting it, then you can manage the Quartz data using supported API.

**Reply by CrazyMoka** [reply]
Thank you!
